include/ccl_params.h

```diff
@@ -44,6 +44,11 @@ typedef struct ccl_spline_params {
 
 extern ccl_spline_params * ccl_splines;
 
+int ccl_get_pk_spline_na(void);
+int ccl_get_pk_spline_nk(void);
+void ccl_get_pk_spline_a_array(int ndout,double* doutput,int *status);
+void ccl_get_pk_spline_lk_array(int ndout,double* doutput,int *status);
```

Comment:
Why are these prototypes here and not in the 2D power spectrum header file?

Reply:
These functions are basically a patch to grab the internal CCL spline parameters and make them visible to python (essentially this gets superseded by your PR, I think). Because they just access the contents of `ccl_spline_params`, I think it's more appropriate for them to be in ccl_params.h

Reply:
Got it. In my PR to make the spline parameters changeable, I moved these structs to `ccl_core.h` and deleted the `ccl_params.h` file. Could we move these prototypes and the function defs to `ccl_core.h` and `ccl_core.c` to avoid the merge conflicts?

Reply:
Done in my last commit

Reply:
These prototypes are still here. Let's delete the params.h file and just use core.h.